kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/NMDbusConnector.java

```diff
@@ -487,14 +487,15 @@ private void enableInterface(String deviceId, NetworkProperties properties, Opti
     
     private void enableInterface(String deviceId, NetworkProperties properties, Device device, NMDeviceType deviceType)
             throws DBusException {
+        NMVersion nmVersion = this.networkManager.getVersion();
         if (Boolean.FALSE.equals(this.networkManager.isDeviceManaged(device))) {
             this.networkManager.setDeviceManaged(device, true);
         }
         String interfaceName = this.networkManager.getDeviceInterface(device);
 
         Optional<Connection> connection = this.networkManager.getAssociatedConnection(device);
         Map<String, Map<String, Variant<?>>> newConnectionSettings = NMSettingsConverter.buildSettings(properties,
-                connection, deviceId, interfaceName, deviceType);
+                connection, deviceId, interfaceName, deviceType, nmVersion);
```

Comment:
I would avoid creating a variable just for this (and think that Sonar would complain) therefore just pass `this.networkManager.getVersion()` to the `buildSettings` method